third_party/bitcoin-core/BUILD.gn

```diff
@@ -109,8 +109,10 @@ static_library("secp256k1") {
     "src/src/secp256k1/src/group_impl.h",
     "src/src/secp256k1/src/hash.h",
     "src/src/secp256k1/src/hash_impl.h",
-    "src/src/secp256k1/src/num.h",
-    "src/src/secp256k1/src/num_impl.h",
+    "src/src/secp256k1/src/precomputed_ecmult.c",
```

Comment:
What's the motivation for this addition in this PR? Is it just a light refactor?

Reply:
That was some refactoring on bitcoin-core / secp256k1 side, so now we should adjust set of flies we include and link with.
For example precomputed_ecmult_gen.h appeared [there](https://github.com/bitcoin/bitcoin/commit/afb7a6fe06a33956ef43429d31f5934448f6e671#diff-315e82b72431171503317c60100564957d5e3b57b03b95932f8be806763d81d1)